test/E2ETest/TyeRunTests.cs

```diff
@@ -240,6 +244,80 @@ public async Task DockerNamedVolumeTest()
             await ProcessUtil.RunAsync("docker", $"volume rm {volumeName}");
         }
 
+        [ConditionalFact]
+        [SkipIfDockerNotRunning]
+        public async Task DockerNetworkAssignmentTest()
+        {
+            using var projectDirectory = CopyTestProjectDirectory("network-test");
```

Comment:
I would actually use the frontend-backend example. That does container to container communication and is written for you already. Forgot to mention that earlier.

Reply:
Let me be more specific, there's a test called `FrontendBackendRunTestWithDocker`. Copy that test and add the custom code to create a network.